fp.txt

```diff
@@ -0,0 +1,17 @@
+[Section]
+Name = Adobe Flash Player 29
+Version = 29.0.0.140
+License = Freeware
+Description = This archive contains Flash Player and its plugins for web browsers (NPAPI and PPAPI).
+Size = 44.9 MiB
+Category = 6
+URLSite = https://www.adobe.com/products/flashplayer.html
+URLDownload = http://svn.reactos.org/packages/fp_29.0.0.140.zip
```

Comment:
There is an official source for these packages - https://forums.adobe.com/thread/2464677

Reply:
All these links are leading to online installer, which does not work on ReactOS. And I would avoid using the online installer if possible anyway.